client/src/App.tsx

```diff
@@ -1,8 +1,28 @@
 import React from 'react';
-import { Home } from './pages/Home';
+import { Home, SplashScreen, WelcomeScreen } from './pages';
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
+import { motion, AnimatePresence } from "framer-motion"
+// import { slideLeft } from './animations'
 
 function App() {
-	return <Home />;
+	return (
+		<BrowserRouter>
+			<AnimatePresence>
+				<motion.div
+					// animate={slideLeft}
```

Comment:
Same as above.

If they are needed, then add a `TODO: un comment this when bla bla bla`